src/Media/ImageCollectionFactoryInterface.php

```diff
@@ -2,22 +2,14 @@
 
 namespace CultuurNet\UDB3\UDB2\Media;
 
-use CultureFeed_Cdb_Data_Media;
+use CultureFeed_Cdb_Item_Base;
 use CultuurNet\UDB3\Media\ImageCollection;
-use CultuurNet\UDB3\Media\Properties\CopyrightHolder;
-use CultuurNet\UDB3\Media\Properties\Description;
 
 interface ImageCollectionFactoryInterface
 {
     /**
-     * @param CultureFeed_Cdb_Data_Media $media
-     * @param Description $fallbackDescription,
-     * @param CopyrightHolder $fallbackCopyright
+     * @param CultureFeed_Cdb_Item_Base $item
      * @return ImageCollection
      */
-    public function fromUdb2Media(
-        CultureFeed_Cdb_Data_Media $media,
-        Description $fallbackDescription,
-        CopyrightHolder $fallbackCopyright
-    );
+    public function fromUdb2Item(CultureFeed_Cdb_Item_Base $item);
```

Comment:
Goede herwerking zodat de logica om de `Media` uit het `Cdb` `Item` te parsen mooi geëncapsuleerd is 👍